optimade/filterparser/tests/testfiles/parse_012.inp

```diff
@@ -1,2 +1 @@
-filter=author="Sąžininga Žąs
-is"
+filter=author="Sąžininga Žąsis"
```

Comment:
This is now identical to `parse_011.inp`, if the intention is to no longer support queries with linebreaks can we remove this test or check that it fails cleanly?

Reply:
You are right! This test started to fail after updating Lark to the latest version. As far as I can see the specification allows NL in the quoted string, so it should pass the test :). Probably the definition of `common.ESCAPED_STRING` has been changed in the new version.

Reply:
I've found [it](https://github.com/lark-parser/lark/commit/d2f55fe3ba7b4bbc95ecdce2c06347cf2314ca4e#diff-4ccafa956f97fcd54ebd361a3bb5aa2e). I also pushed the modifications for all the previous grammar versions.